modules/fargate/README.md

```diff
@@ -37,36 +38,36 @@ No modules.
+| <a name="input_create_fargate_pod_execution_role"></a> [create\_fargate\_pod\_execution\_role](#input\_create\_fargate\_pod\_execution\_role) | Controls if the the IAM Role that provides permissions for the EKS Fargate Profile should be created. | `bool`         | `true`  |    no    |
+| <a name="input_fargate_pod_execution_role_name"></a> [fargate\_pod\_execution\_role\_name](#input\_fargate\_pod\_execution\_role\_name)       | The IAM Role that provides permissions for the EKS Fargate Profile.                                   | `string`       | `null`  |    no    |
+| <a name="input_fargate_profiles"></a> [fargate\_profiles](#input\_fargate\_profiles)                                                          | Fargate profiles to create. See `fargate_profile` keys section in README.md for more details          | `any`          | `{}`    |    no    |
+| <a name="input_iam_path"></a> [iam\_path](#input\_iam\_path)                                                                                  | IAM roles will be created on this path.                                                               | `string`       | `"/"`   |    no    |
+| <a name="input_permissions_boundary"></a> [permissions\_boundary](#input\_permissions\_boundary)                                              | If provided, all IAM roles will be created with this permissions boundary attached.                   | `string`       | `null`  |    no    |
+| <a name="input_subnets"></a> [subnets](#input\_subnets)                                                                                       | A list of subnets for the EKS Fargate profiles.                                                       | `list(string)` | `[]`    |    no    |
+| <a name="input_tags"></a> [tags](#input\_tags)                                                                                                | A map of tags to add to all resources.                                                                | `map(string)`  | `{}`    |    no    |
 
 ## Outputs
 
-| Name | Description |
-|------|-------------|
-| <a name="output_aws_auth_roles"></a> [aws\_auth\_roles](#output\_aws\_auth\_roles) | Roles for use in aws-auth ConfigMap |
-| <a name="output_fargate_profile_arns"></a> [fargate\_profile\_arns](#output\_fargate\_profile\_arns) | Amazon Resource Name (ARN) of the EKS Fargate Profiles. |
-| <a name="output_fargate_profile_ids"></a> [fargate\_profile\_ids](#output\_fargate\_profile\_ids) | EKS Cluster name and EKS Fargate Profile names separated by a colon (:). |
-| <a name="output_iam_role_arn"></a> [iam\_role\_arn](#output\_iam\_role\_arn) | IAM role ARN for EKS Fargate pods |
-| <a name="output_iam_role_name"></a> [iam\_role\_name](#output\_iam\_role\_name) | IAM role name for EKS Fargate pods |
+| Name                                                                                                 | Description                                                              |
```

Comment:
what has done those changes?

this part of file should be autogenerated via pre commit hooks